pkg/apis/core/validation/validation.go

```diff
@@ -4058,14 +4057,24 @@ func ValidateAppArmorPodAnnotations(annotations map[string]string, spec *core.Po
 			allErrs = append(allErrs, field.Invalid(fldPath.Key(k), containerName, "container not found"))
 		}
 
-		if err := apparmor.ValidateProfileFormat(p); err != nil {
+		if err := ValidateAppArmorProfileFormat(p); err != nil {
 			allErrs = append(allErrs, field.Invalid(fldPath.Key(k), p, err.Error()))
 		}
 	}
 
 	return allErrs
 }
 
+func ValidateAppArmorProfileFormat(profile string) error {
```

Comment:
nit: do we wanna make this helper private?

Reply:
used from `pkg/security/apparmor`, so not now?